src/components/molecules/EntityGrid/EntityGridItemPure.tsx

```diff
@@ -15,85 +15,99 @@ import Colors from '@styles/Colors';
-                  : null}
-              </Text>
-            </Tooltip>
-          </ItemColumn>
-        </ItemRow>
-        <RowsWrapper>
-          <ItemRow $flex={1}>
-            {dataItem?.labels ? (
-              <LabelsList labels={dataItem?.labels} shouldSkipLabels howManyLabelsToShow={2} />
-            ) : null}
-          </ItemRow>
-          <ItemRow $flex={1}>
-            <StyledMetricItem>
-              <Text className="small uppercase" color={Colors.slate500}>
-                pass/fail ratio
-              </Text>
-              <Text className="big regular">
-                {metrics?.passFailRatio ? `${metrics?.passFailRatio.toFixed(2)}%` : '0%'}
-              </Text>
-            </StyledMetricItem>
-            <StyledMetricItem>
-              <Text className="small uppercase" color={Colors.slate500}>
-                p50
-              </Text>
-              <Text className="big regular">
-                {metrics?.executionDurationP50ms ? formatDuration(metrics?.executionDurationP50ms / 1000) : '-'}
-              </Text>
-            </StyledMetricItem>
-            <StyledMetricItem>
-              <Text className="small uppercase" color={Colors.slate500}>
-                failed executions
-              </Text>
-              <Text className="big regular">{metrics?.failedExecutions || '-'}</Text>
-            </StyledMetricItem>
-            <StyledMetricItem>
-              <MetricsBarChart data={executions} chartHeight={38} barWidth={6} />
-            </StyledMetricItem>
+    return (
+      <ItemWrapper onClick={click} ref={ref} data-test={dataTestValue}>
+        <DetailsWrapper>
+          <ItemRow $flex={0}>
+            <ItemColumn $isStretch>
+              <StatusIcon status={status} />
+              {dataItem?.type ? <ExecutorIcon type={dataItem?.testIcon} /> : null}
+              <div style={{overflow: 'hidden', flex: 1, display: 'flex'}}>
+                <Text className="regular big" ellipsis title={dataItem?.name}>
+                  {dataItem?.name}
+                </Text>
+              </div>
+            </ItemColumn>
+            <ItemColumn>
+              <Tooltip
+                title={latestExecution?.startTime ? formatExecutionDate(new Date(latestExecution?.startTime)) : null}
+                placement="bottomRight"
+              >
+                <Text color={Colors.slate200}>
+                  {latestExecution?.startTime
+                    ? displayTimeBetweenDates(new Date(), new Date(latestExecution?.startTime)).long
+                    : null}
+                </Text>
+              </Tooltip>
+              <DotsDropdown
+                items={[
+                  {
+                    key: 1,
+                    label: <span onClick={event => onAbortAllExecutionsClick(event)}>Abort all executions</span>,
+                  },
+                ]}
+                trigger={['hover']}
```

Comment:
Should there actually be a different trigger than `['click']`? I recall that we've been explicitly changing all dropdowns to be on click instead of hover.

@fivenp, could you advice? It's on each (clickable) element of tests/test suites list.

Reply:
For now I think all our burger menus should be on click – I can't think of any case where we want them on hover